pmlc/dialect/stripe/agginit_pass.cc

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019, Intel Corporation
```

Comment:
Overall this pass is hard to follow.  I'm not sure if it's a design flaw in Stipe IR or something else, but I feel like this should be less complex.  Nothing needs to be done about this, but it's interesting to read this code.